.github/workflows/haskell-ci.yml

```diff
@@ -71,6 +88,11 @@ on:
         type: boolean
         required: false
         default: true
+      sdist-targets:
+        description: Targets passed to `cabal sdist`.
```

Comment:
Perhaps say that if this is not specified, then this will default to all of the targets in the current package. (`default: ""` doesn't really make this obvious.)